teradatalabs/mapr52-base/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM teradatalabs/centos6-java8-oracle
+MAINTAINER Teradata Docker Team <docker@teradata.com>
+
+
+#Add repo for mapr
+ADD files/maprtech.repo /etc/yum.repos.d/maprtech.repo
+RUN yum update -y
```

Comment:
If possible, combine all the software installation with this step and do this first.